deploy/docker-ephemeral/build/README.md

```diff
@@ -0,0 +1,29 @@
+A makefile that uses docker.io, and qemu-user-static to build dependencies for our integration tests.
+
+Builds docker images for multiple architectures. allows for -j to build multiple images at once.
+
+# Setup
+
+## Docker
+docker.io is only available from debian testing or unstable.
+`apt-get install docker.io`
+
+after installing docker-io, add the user you are running as to the docker group, and restart X.
+
+once you've logged in again,
+# docker login --username=$(USERNAME)
+
+## qemu
+`apt-get install qemu-user-static`
+`sudo service binfmt-support start`
+`cat /proc/sys/fs/binfmt_misc/qemu-arm`
+make sure that you see 'flags: OCF'.
```

Comment:
On fedora, the following seems sufficient:

```
sudo dnf install -y qemu-user-static
# ensure the following shows flags: F:
cat /proc/sys/fs/binfmt_misc/qemu-arm
```

Reply:
Added, and broke out to a separate troubleshooting section.